From 8c9f6e0377ed5c1dc7b137e4a3e89129f6176275 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Mon, 1 Aug 2011 14:09:33 -0700
Subject: [PATCH] Don't fallback to wsfb or fbdev on Solaris

We don't ship either one, so don't waste time and make confusing log
entries trying to load them.

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 hw/xfree86/common/xf86AutoConfig.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/hw/xfree86/common/xf86AutoConfig.c b/hw/xfree86/common/xf86AutoConfig.c
index 7dc3189..76a9109 100644
--- a/hw/xfree86/common/xf86AutoConfig.c
+++ b/hw/xfree86/common/xf86AutoConfig.c
@@ -300,6 +300,7 @@ listPossibleVideoDrivers(char *matches[], int nmatches)
 #endif
     }
 
+#if !defined(sun)
     /* Fallback to platform default frame buffer driver */
     if (i < (nmatches - 1)) {
 #if !defined(__linux__) && defined(__sparc__)
@@ -308,6 +309,7 @@ listPossibleVideoDrivers(char *matches[], int nmatches)
 	matches[i++] = xnfstrdup("fbdev");
 #endif
     }
+#endif /* !sun */
 }
 
 /* copy a screen section and enter the desired driver
-- 
1.7.3.2

